_posts/2017-05-02-newsletter.md

```diff
@@ -0,0 +1,222 @@
+  -  Path handling on windows has been improved (**@aignas**, https://github.com/neovim/neovim/pull/7349)
+- For Linux users:
+  - You can now download AppImage packages (**@AdnoC**, https://github.com/neovim/neovim/pull/6638)
+  - Function keys now work in the terminal UI (**@rjmill**, https://github.com/neovim/neovim/pull/5014)
+  - Very long lines do not crash Neovim anymore (**@Grimy**, https://github.com/neovim/neovim/pull/3527)
+  - `cquit` now takes an optional error code as argument (**@joshleeb**, https://github.com/neovim/neovim/pull/7336)
+  - True colors in tmux have been fixed (**@DarkDefender**, https://github.com/neovim/neovim/pull/7100)
+  
+
+
+#### 0.2.1-dev for Developers
+- Changes:
+  - External UI's now should use the `FocusGained` event instead of sending the `<FocusGained>` pseudokey (**@justinmk**, https://github.com/neovim/neovim/pull/7221)
+- Neovim sources are now analyzed with PVS (**@ZyX-l**, https://github.com/neovim/neovim/pull/6493)
+- External UIs can now draw
+  - ... the tabline (**@dzhou121**, https://github.com/neovim/neovim/pull/6583)
+  - ... the command line (**@dzhou121**, **@bfredl**, https://github.com/neovim/neovim/pull/6162)
+  - ... the wildmenau (**@dzhou121**, **@bfredl**, https://github.com/neovim/neovim/pull/7454)
+- Lua is gaining momentum:
+  - The interface has been implemented (**@ZyX-I**, https://github.com/neovim/neovim/pull/4411)
+  - Lua files can now be required from `lua/` (**@ZyX-l**, https://github.com/neovim/neovim/pull/6789)
+  - Call lua directly from the API (**@bfredl**, https://github.com/neovim/neovim/pull/6704)
+- Programmatically get information about mappings via  `get_keymap` (**@tjdevries**, https://github.com/neovim/neovim/pull/6236)
+ - You can now retrieve highlights `using nvim_get_hl_by_name/by_id` (**@teto**, https://github.com/neovim/neovim/pull/7082)
+ - `bufhl` can now be used to create new highlighting groups (**@bfredl**, https://github.com/neovim/neovim/pull/7414)
+ - External UI's can get menus from  `menu_get` (**@teto**, https://github.com/neovim/neovim/pull/6322)
+    - `menu_get` pretty-prints special chars, making it possible to feed its results back into e.g. `nvim_input` (**@teto**, **@KillTheMule**, https://github.com/neovim/neovim/pull/7340)
+ - RPC clients can connect to a socket using `sockconnect` (**@bfredl**, https://github.com/neovim/neovim/pull/6594)
+ - `serverstart` now uses `uv_getaddrinfo()`, bringing IPv6 support (**@mhinz**, https://github.com/neovim/neovim/pull/6680)
+ - You can now define several functions in one `:execute` call (**@ZyX-I**, https://github.com/neovim/neovim/pull/6914)
+ 
+ 
+  
+### The not-so-near future: Upcoming
+
+For an overview of planned features, goals and ideas for Neovim head to the [road
+map](https://neovim.io/roadmap/). Some noteworthy upcoming PRs are:
+
+- Floating Windows in the Terminal (**@bfredl**, https://github.com/neovim/neovim/pull/6619)
```

Comment:
"in the Terminal" can be dropped, https://github.com/neovim/neovim/pull/6619 is now also supporting external uis.